src/main/java/org/opentripplanner/middleware/models/User.java

```diff
@@ -13,6 +13,8 @@
     public String email;
     // FIXME: This will eventually reference the Auth0 user ID and not a randomly generated UUID.
     public String auth0UserId = UUID.randomUUID().toString();
-    /** Only store trip planning requests/results if a user has explicitly opted in. */
-    public boolean storeResults;
+    public OpenTripPlannerOptions openTripPlannerOptions;
```

Comment:
By storing both `openTripPlannerOptions` and `adminOptions` in the same class, it looks like we are mashing together the users of OTP and the users of the admin dashboard and saying that a regular OTP user could have access to the admin dashboard... which one is it? I think both kinds of users should be separate (they can derive from a common user class though).